crates/scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Major
 
+- Add `radio` module
```

Comment:
This is not major since it doesn't change the API of the scheduler. It's a minor change:

```
- Support `radio::ble`
```